resources/install.ps1

```diff
@@ -24,18 +24,18 @@ if (-not (Test-Path $sp_dot_dir)) {
 }
 
 Write-Host "Downloading..." -ForegroundColor "Green"
-Invoke-WebRequest -Uri "https://github.com/CharlieS1103/spicetify-marketplace/archive/refs/heads/main.zip" -UseBasicParsing -OutFile "${HOME}/spicetify-cli/CustomApps/spicetify-marketplace.zip"
+Invoke-WebRequest -Uri "https://github.com/spicetify/spicetify-marketplace/archive/refs/heads/dist.zip" -UseBasicParsing -OutFile "${HOME}/spicetify-cli/CustomApps/marketplace.zip"
 
 Write-Host "Unzipping and installing..." -ForegroundColor "Green"
-Expand-Archive -Path "${HOME}/spicetify-cli/CustomApps/spicetify-marketplace.zip" -DestinationPath "${HOME}/spicetify-cli/CustomApps/" -Force
-Remove-Item -Path "${HOME}/spicetify-cli/CustomApps/spicetify-marketplace.zip" -Force
-if (Test-Path -Path "${HOME}/spicetify-cli/CustomApps/spicetify-marketplace") {
-  Write-Host "spicetify-marketplace was already found! Updating..." -ForegroundColor "Cyan"
-  Remove-Item -Path "${HOME}/spicetify-cli/CustomApps/spicetify-marketplace" -Force -Recurse
+Expand-Archive -Path "${HOME}/spicetify-cli/CustomApps/marketplace.zip" -DestinationPath "${HOME}/spicetify-cli/CustomApps/" -Force
+Remove-Item -Path "${HOME}/spicetify-cli/CustomApps/marketplace.zip" -Force
+if (Test-Path -Path "${HOME}/spicetify-cli/CustomApps/marketplace") {
+  Write-Host "marketplace was already found! Updating..." -ForegroundColor "Cyan"
+  Remove-Item -Path "${HOME}/spicetify-cli/CustomApps/marketplace" -Force -Recurse
 }
-Rename-Item -Path "${HOME}/spicetify-cli/CustomApps/spicetify-marketplace-main" -NewName "spicetify-marketplace" -Force
-Copy-Item -Path "${HOME}/spicetify-cli/CustomApps/spicetify-marketplace" -Destination $sp_dot_dir -Recurse -Force
-spicetify config custom_apps spicetify-marketplace
+Rename-Item -Path "${HOME}/spicetify-cli/CustomApps/marketplace-dist" -NewName "marketplace" -Force
+Copy-Item -Path "${HOME}/spicetify-cli/CustomApps/marketplace" -Destination $sp_dot_dir -Recurse -Force
+spicetify config custom_apps marketplace
```

Comment:
```suggestion
spicetify config custom_apps spicetify-marketplace-
spicetify config custom_apps marketplace
```
Remove the old marketplace from file as it applies both if it is previously installed
![image](https://user-images.githubusercontent.com/77577746/172033843-11537250-675e-45cc-b2e2-3eb040b9e65d.png)